src/components/ProfileOverview/index.js

```diff
@@ -36,7 +37,7 @@ const ProfileOverview = ({
                         </div>
                     }
                     {
-                        loggedIn &&
+                        loggedIn && loggedInUserId === profileOwner._id &&
```

Comment:
Here, if this component is used in ```ProfileScreen```, then we never display this private part, even if the logged-in user is the profile owner.

I suggest you add one more argument to this component ```private=false```, and in ```MyProfileScreen``` you  use this component with ```private=true```